stable/mission-control/README.md

```diff
@@ -105,6 +105,16 @@ Once you have a new chart version, you can update your deployment with
 helm upgrade mission-control jfrog/mission-control
 ```
 
+If mission-control was installed without providing a value to postgresql.postgresPassword (a password was autogenerated), follow these instructions:
```

Comment:
JFMC already has a *Auto generated passwords* section that covers this.

Reply:
Removed